library/model/user.py

```diff
@@ -1,82 +1,86 @@
-from typing import Optional
-from library.model.book import Book, BorrowedBook
-from library.model.genre import Genre
+from typing import NamedTuple
+from library.model.book import Book, BookBorrowed
+from library.model.reading_creadits import ReadingCreditsPerGenre
+from library.payment.invoice import Invoice
 from library.persistence.storage import LibraryRepository
 
 
-class User:
-
+class PersonalData(NamedTuple):
     email: str
-    borrowed_books: list[BorrowedBook]
-    read_books: list[Book]
-    invoices: list
     firstname: str
```

Comment:
I would not extract personal data class, as it has 1 to 1 relation to user and creates more overhead